docs/aws/provisioner-launch-templates-options.md

```diff
@@ -296,3 +328,10 @@ leverage and/or work smoothly with
 [CAPI](https://github.com/kubernetes-sigs/cluster-api). We will
 address this in a separate document.
 
+## Launch Template Names
+
+We could support something like:
+
+- `node.k8s.aws/launch-template-name`: (optional) name of launch template
```

Comment:
Optional: Maybe call out the problems (or work needed to support this)?  For example if a provisioner used launch-template-id but the pod had an override with the launch-template-name, then two nodes would be provisioned. Maybe this is just a scenario that users shouldn't do and we shouldn't try to help too much.

Reply:
I can address in a follow up